### DISCUSSION TITLE
How to set custom title for not found pages in Remix?

### DISCUSSION THREAD

**Discussion by runoncedev** [remix-run/remix#3906] [Q&A]
I'm setting up my 404 page on a Remix app but I'm struggling on how to set the `<title>` meta tag for such pages.

In my `root.tsx` I have a `MetaFunction` and a `CatchBoundary`:

```
export const meta: MetaFunction = () => {
  return {
    charset: "utf-8",
    title: "New Remix App",
    viewport: "width=device-width,initial-scale=1",
  };
};

export function CatchBoundary() {
  const caught = useCatch();

  return (
    <html lang="en">
      <head>
        <Meta />
        <Links />
      </head>
      <body>
        <div>
          <h1>Caught</h1>
          <p>Status: {caught.status}</p>
          <pre>
            <code>{JSON.stringify(caught.data, null, 2)}</code>
          </pre>
        </div>
      </body>
    </html>
  );
}
```

I tried using the `data` parameter of the `MetaFunction` but it's not guaranteed that not having `data` means that it is a not found response catch.

I went through several pages looking for an answer, including:

* https://remix.run/docs/en/v1/guides/not-found
* https://remix.run/docs/en/v1/api/conventions#catchboundary

This is the repository:

* https://github.com/oncet/blog

**Comment by lili21** [comment]
you could just define your own meta function in your 404 page/route

**Reply by lili21** [reply]
it will overwrite the root title

**Reply by runoncedev** [reply]
I just wanted to use the root 404 page for all routes and set the title to something like "Page not found", isn't this possible?

**Reply by kiliman** [reply]
Sure. You don't need the `<Meta/>`. Just render a `<title/>`
```js
return (
    <html lang="en">
      <head>
        <title>Page not found</title>
        <Links />
      </head>
```

**Reply by kiliman** [reply]
That's the beauty of Remix. You control the entire HTML.

**Reply by runoncedev** [reply]
I will try that!

Although I'm a bit confused on why we need `<Meta />` if we can simply use normal HTML meta tags 🤔

**Reply by kiliman** [reply]
The Remix `<Meta/>` component is used to output meta/title based on the results from each routes `meta` function. A child route can export a `meta` function to change the title, for example.

**Reply by runoncedev** [reply]
It's kinda awkward that if I don't use the `<Meta />` component I need to provide all other meta tags manually.

Would be awesome if it allowed receiving title through some prop.
